fsspec/implementations/local.py

```diff
@@ -77,25 +77,21 @@ def info(self, path, **kwargs):
         return result
 
     def copy(self, path1, path2, **kwargs):
-        """ Copy within two locations in the filesystem"""
```

Comment:
These were identical to the parent class's. Sphinx will pick them up automatically.